src/helpers/graphql.ts

```diff
@@ -28,8 +28,20 @@ export async function proposalQuery(id) {
           id: true,
           name: true
         }
+      },
+      votes: {
+        __args: {
+          first: 10000,
+          where: {
+            proposal: id
+          }
+        },
+        id: true,
+        voter: true,
+        created: true,
+        choice: true
       }
     }
   );
-  return response.proposal;
+  return { proposal: response.proposal, votes: response.votes };
```

Comment:
You can do this
```suggestion
  return response;
```
Or this directly
```js
return await subgraphRequest(...
```

Reply:
But then we need to do more restructuring afterwards. Because the response will be `response: {votes: {}, proposal: {}}`

Reply:
Isn't the same than what you return already?

Reply:
Yes 😄 somehow it made sense before when I did each query separately.